service/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -30,9 +31,12 @@ public class DatarepoService {
   public static final String READER_ROLE_NAME = "reader";
   public static final String DISCOVERER_ROLE_NAME = "discoverer";
 
-  private static final List<String> OWNER_ROLES = List.of(ADMIN_ROLE_NAME, STEWARD_ROLE_NAME);
-  private static final List<String> READER_ROLES =
-      List.of(ADMIN_ROLE_NAME, STEWARD_ROLE_NAME, READER_ROLE_NAME, DISCOVERER_ROLE_NAME);
+  public static final Map<String, DatasetAccessLevel> SNAPSHOT_ROLE_TO_DATASET_ACCESS_LEVEL =
```

Comment:
Can this be `private`? Also I prefer shorter names when meaning aren't ambiguous. Since we only deal with snapshot roles here I think `ROLE_TO_DATASET_ACCESS_LEVEL` or even `ROLE_TO_DATASET_ACCESS` would be fine.